src/intro.md

```diff
@@ -82,3 +82,74 @@ Open-source and freely distributable [assets](https://app.ignitionrobotics.org/f
 ## Jump in, the water is fine!
 
 So now you have an idea of what RMF is all about, it's time to jump in. We would suggest if you have not already that you take the time to review the [RMF Demos](https://github.com/osrf/rmf_demos) repository and if you want a really quick overview of RMF then take a look at this [Mock Airport Terminal video demo](https://vimeo.com/405803151) (Short film Oscar nominations most welcome). We hope you find RMF to be a useful tool to help you scale your robot deployments and operations and we look forward to the many improvements and contributions to come!
+
+### Installation of the RMF Essentials
+
+As RMF makes heavy use of the ROS and Gazebo-Ignition ecosystem, we will have to setup their repositories as the first steps of the RMf installation process. The current binary release targets ubuntu bionic, therefore make sure you match the system and version before proceeding with the installation.
+
+#### Setup Locale
+
+Make sure you have a locale which supports UTF-8. If you are in a minimal environment, such as a docker container, the locale may be something minimal like POSIX. We test with the following settings. It should be fine if you’re using a different UTF-8 supported locale.
```

Comment:
```suggestion
Make sure you have a locale which supports UTF-8. If you are in a minimal environment, such as a docker container, the locale may be something minimal like POSIX. It should be fine if you’re using a different UTF-8 supported locale. You can test this with the following steps:
```

Reply:
This one means that **we** are actually running tests with these settings, it might work without them but it's better to set this up to avoid issues. This is actually a direct copy of the same step in the `ROS` installation instructions [here](https://index.ros.org/doc/ros2/Installation/Crystal/Linux-Install-Debians/#id2).

Reply:
Thanks for clarifying!